lib/matplotlib/testing/decorators.py

```diff
@@ -135,6 +136,8 @@ def copy_baseline(self, baseline, extension):
             with contextlib.suppress(OSError):
                 os.remove(expected_fname)
             try:
+                if 'microsoft' in uname().release.lower():
+                    raise OSError  # On WSL, symlink breaks silently
                 os.symlink(orig_expected_path, expected_fname)
```

Comment:
Could this be made more general by doing something like:

```suggestion
                os.symlink(orig_expected_path, expected_fname)
                # Stat will raise an OSError if missing
                os.stat(expected_fname)
```

That would remove the need to check for specific strings as markers while ensuring the behavior we want across all platforms, would it not?

Reply:
(Technically stat raises `FileNotFoundError`, but that is an OSError subclass and OSError is what matters here)

Reply:
Ah, fair enough, if this is really that specific, then not too fussed about the string check. I was wondering if the mechanism was something along the lines of using a file system that doesn't support symlinks but through linux which mostly _does_ support symlinks (though I'd _hope_ that generally speaking that would raise an OSError)